style.css

```diff
@@ -364,3 +364,77 @@ li {
   font-weight: 500;
   font-size: 20px;
 }
+
+.contact {
+  background-color: #6070ff;
+  background-image: url(./img/contact\ form\ shapes-mobile.png);
+  background-position: right;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  border-top-left-radius: 100px;
+}
+
+.contact h2 {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  text-align: center;
+  color: #fff;
+  padding-top: 103px;
+  padding-bottom: 12px;
+}
+
+.contact p {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  color: #ebebff;
+  text-align: center;
+  padding: 0 24px 30px 24px;
+}
+
+#username,
+#user-email,
+#text-area {
+  width: 90%;
+  margin: 20px 24px;
+  height: 2rem;
+  border-radius: 8px;
+  border: 1px solid #cfd8dc;
+}
```

Comment:
- [ ] :thinking: I think It would be better to stick to the design as much as possible, kindly make sure to remove all the border radius, the margin and add to the height of input fields here  :
  - Your design:
![area](https://user-images.githubusercontent.com/97834160/206041676-c3a702fa-eb52-4e57-8dab-b7deb425d60f.png)

  
  - Figma:

    
![figFor](https://user-images.githubusercontent.com/97834160/206042273-49c7dd1f-d802-46b0-8297-d3033d34fd83.png)